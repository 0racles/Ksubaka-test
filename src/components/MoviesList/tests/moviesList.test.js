import { MoviesList } from '../index';

const props = {
  onClick: (f) => f,
  items: [],
};

describe('Movies List', () => {
  it('should match snapshot', () => {
    const renderedItem = shallow(<MoviesList {...props} />);
    expect(renderedItem).toMatchSnapshot();
  });
});
