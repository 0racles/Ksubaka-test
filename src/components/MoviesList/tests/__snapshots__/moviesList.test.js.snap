// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies List should match snapshot 1`] = `
<div>
  <.sc-bdVaJa
    className="itemsList-ul"
  >
    <p
      id="items-missing"
    >
      No items found.
    </p>
  </.sc-bdVaJa>
</div>
`;
